04-set-up.Rmd

```diff
@@ -384,41 +383,43 @@ towny %>%
             area_median = median(land_area_km2))
 ```
 
-In the code below, we calculate the mean and median of the variable `api00` using `dstrata`. Note the similarity in the syntax. When we dig into the {srvyr} functions later, we will show that the outputs share a similar structure. Each group (if present) generates one row of output, but with additional columns. By default, the standard error of the statistic is also calculated in addition to the statistic itself.
+In the code below, we calculate the mean and median of the variable `api00` using `apistrat_des`. Note the similarity in the syntax. However, the standard error of the statistic is also calculated in addition to the statistic itself.
 
 ```{r}
 #| label: setup-srvyr-examp
-dstrata %>%
+apistrat_des %>%
   summarize(api00_mean = survey_mean(api00),
             api00_med = survey_median(api00))
 ```
 
-The functions in {srvyr} also play nicely with other tidyverse functions. For example, if we wanted to select columns with shared characteristics, we can use {tidyselect} functions such as `starts_with()`, `num_range()`, etc. In the examples below, we use a combination of `across()` and `starts_with()` to calculate the mean of variables starting with "population" in the `towny` data frame and those beginning with `api` in the `dstrata` survey object.
+The functions in {srvyr} also play nicely with other tidyverse functions. For example, if we wanted to select columns with shared characteristics, we can use {tidyselect} functions such as `starts_with()`, `num_range()`, etc. In the examples below, we use a combination of `across()` and `starts_with()` to calculate the mean of variables starting with "population" in the `towny` data frame and those beginning with `api` in the `apistrat_des` survey object.
 
 ```{r}
 #| label: setup-dplyr-select
 towny %>%
-  summarize(across(starts_with("population"), ~mean(.x, na.rm=TRUE)))
+  summarize(across(starts_with("population"), 
+                   ~mean(.x, na.rm=TRUE)))
 ```
 
 ```{r}
 #| label: setup-srvyr-select
-dstrata %>%
-  summarize(across(starts_with("api"), survey_mean))
+apistrat_des %>%
+  summarize(across(starts_with("api"), 
+                   survey_mean))
```

Comment:
This code does not use the ~ and .x in the function, but the mean in the code above does.  Should these be consistent?